src/Compiler/SyntaxTree/ParseHelpers.fs

```diff
@@ -1159,7 +1159,7 @@ let mkSynField
 
             let mType, mStart =
                 idOpt
-                |> Option.map _.idRange
+                |> Option.map (fun x -> x.idRange)
```

Comment:
Why? Is this style preference change or is there semantic or perf difference?

Reply:
No, it was failing me when building directly in VS (complaining about older language feature, since this is an F#8 feature) and I did not want to spend more than 30s on it.
I am not running build script for every single small change, this might be the reason.